### PR TITLE
Add known issues to Teams App caching page

### PR DESCRIPTION
**Pull request by ZimnitskiYury**
This update includes the addition of a 'Known Issue' section to inform users that app caching is currently not supported in the new Teams application. This is an important note for developers and users who rely on caching for performance.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/tabs/how-to/app-caching.md](https://github.com/MicrosoftDocs/msteams-docs/blob/22176cd567f198465bcfdf3ecaa040f46a8269de/msteams-platform%2Ftabs%2Fhow-to%2Fapp-caching.md) | 89 | 16 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/55033fdb-7d56-4ca2-86f6-c46ffba8783e) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/tabs/how-to/app-caching.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c85fb6a089629a2ec70ee8097295afe1dbdc4ab0/msteams-platform%2Ftabs%2Fhow-to%2Fapp-caching.md) | 89 | 16 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/dbf8dcce-9b3a-4bf5-b7ea-be111dc47c92) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/tabs/how-to/app-caching.md](https://github.com/MicrosoftDocs/msteams-docs/blob/02f31cfe20c31348988c35d2799bc3c77c41bc99/msteams-platform%2Ftabs%2Fhow-to%2Fapp-caching.md) | 89 | 16 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3a3fcb0a-ca6c-4aa4-8172-818002b88f42) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-sdhakshina — CHANGES_REQUESTED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/tabs/how-to/app-caching.md](https://github.com/MicrosoftDocs/msteams-docs/blob/59ada36f2de74cf409e238f831b5c1297d67518b/msteams-platform%2Ftabs%2Fhow-to%2Fapp-caching.md) | 88 | 16 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4f845d43-a244-4d23-88e4-1415279e0300) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/tabs/how-to/app-caching.md [hunk 1]
```diff
@@ -116,6 +116,8 @@ The following are the limitations for app caching:
   * `readyToUnload`
   * `getConfig/getSettings`
 
+* App caching is not supported in the [new Teams client](https://learn.microsoft.com/microsoftteams/platform/resources/teams-updates#known-issues)
```
Comment (v-sdhakshina) [inline msteams-platform/tabs/how-to/app-caching.md:119]:
```suggestion
* App caching isn't supported in the [new Teams client](../resources/teams-updates.md#known-issues).
```

**Review by v-sdhakshina — CHANGES_REQUESTED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/tabs/how-to/app-caching.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b8a3f8467060f4cc10035b90bdbf1455960a70a0/msteams-platform%2Ftabs%2Fhow-to%2Fapp-caching.md) | 90 | 15 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/852c8073-cc6c-4abe-8a8a-04165443c1e3) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by Wajeed-msft — APPROVED** [review]

**Review by v-sdhakshina — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/tabs/how-to/app-caching.md](https://github.com/MicrosoftDocs/msteams-docs/blob/fb048ce6ee51d934cadb971671c9f473829d40a3/msteams-platform%2Ftabs%2Fhow-to%2Fapp-caching.md) | 90 | 15 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7387b93c-df46-4023-9ec0-00224c146775) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/tabs/how-to/app-caching.md](https://github.com/MicrosoftDocs/msteams-docs/blob/704c9299b3a6b9f9a6af9527f3168b0d7c7a4321/msteams-platform%2Ftabs%2Fhow-to%2Fapp-caching.md) | 90 | 15 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/bb53763f-2bbf-4b47-a39d-aa4b235b03e0) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by surbhigupta12 — APPROVED** [review]

**Review by v-deabira — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/tabs/how-to/app-caching.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8ab9a6d319960a4bc294ad64c99d96ddb471d861/msteams-platform%2Ftabs%2Fhow-to%2Fapp-caching.md) | 90 | 15 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/de96348f-bcd0-4f09-a814-bd77f86b86f8) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Pull request merged**